source/newcommitter.md

```diff
@@ -140,178 +150,54 @@ with no holds barred. Keep the discussion concise. The praise part can
 be done later in public. Keep in mind, however, that if the member becomes
 a PMC member later, they will have access to this discussion.
 
-Let the Vote thread run for one week.
+### Vote
+
+If the proposed candidate seems to be received positively by a majority
+of those responding, it's time to start a vote.
 
-A positive result is achieved by **Consensus Approval**: at least 3 +1
-votes and no vetoes.
+You can use [this template to start the
+vote](templates/committer-vote.txt). In some projects, new committers
+are automatically also made PMC members. If this is the case in your
+project, you can use [this template to start the
+vote](committer-pmc-vote.txt).
+
+Start a separate [VOTE] thread for each new person. This makes it much easier
+to review the email archives.
 
+Let the Vote thread run for one week. A positive result is achieved by 
+**Consensus Approval**: at least 3 +1 votes and no vetoes.
```

Comment:
Just to clarify - this is the same as "votes on code modification"  https://www.apache.org/foundation/voting.html#votes-on-code-modification right?  - since veto is mentioned, that should likely be the same.

Maybe we should update the voting page - and we should mention "Voting on new committers" as a separate entry there - I am happy to propose it also as a PR to the main site, but that seems like a more involved change that requires VP legal review maybe ? Or can I just make PR there adding this "voting type" ? Or maybe you think it's not needed?

Reply:
According to https://apache.org/foundation/voting this is not a veto-able vote, but is a simple majority vote.